p2p/peer/serverpeer.go

```diff
@@ -16,7 +16,7 @@ type ServerPeer struct {
 	// The following chans are used to sync blockmanager and server.
 	*Peer
 	TxProcessed     chan struct{}
-	BlockProcessed  chan struct{}
+	BlockProcessed  chan int
```

Comment:
same as above, use user-defined type instead of raw `int` type for code readability and Please document the design change & purpose because the meaning of the `BlockProcessed` channel has been changed in the context.

Reply:
https://github.com/Qitmeer/qitmeer/pull/255/commits/b8470458f82c4b88dfa42aa2e84fa3b7026b8f20